### ISSUE TITLE
IllegalStateException

### ISSUE THREAD

**Issue by msya** [yodle/android-kotlin-demo#15]
I encountered this crash in the app.

```
java.lang.IllegalStateException: Cannot start this animator on a detached view!
at android.view.RenderNode.addAnimator(RenderNode.java:812)
at android.view.RenderNodeAnimator.setTarget(RenderNodeAnimator.java:300)
at android.view.RenderNodeAnimator.setTarget(RenderNodeAnimator.java:282)
at android.animation.RevealAnimator.<init>(RevealAnimator.java:37)
at android.view.ViewAnimationUtils.createCircularReveal(ViewAnimationUtils.java:55)
at com.yodle.android.kotlindemo.extension.ViewExtensionsKt$circularReveal$1.run(ViewExtensions.kt:47)
at android.os.Handler.handleCallback(Handler.java:739)
at android.os.Handler.dispatchMessage(Handler.java:95)
at android.os.Looper.loop(Looper.java:148)
at android.app.ActivityThread.main(ActivityThread.java:5417)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

**Comment by dsn5ft** [comment]
Interesting, it looks like the animation is being calculated before the view is fully inflated (https://github.com/wordpress-mobile/WordPress-Android/issues/2245). I'll put the animation within a view.post().

Does it happen every time you open the page or just sometimes?

**Comment by msya** [comment]
It happens sometimes. The crash can be reproduced if you click on a repo, while the detail page is loading go back to the search results, the crash will occur.

**Comment by dsn5ft** [comment]
Hmm, that makes sense but for some reason I can't reproduce it. I tried opening and closing the page a bunch of times and increasing the delay so that it definitely happens after the page is closed, but I can't get it to crash.

I'm wondering if there's anything that can be done to fix the problem besides try-catching it. It looks like View.isAttachedToWindow() was added in API 19 but the project's min sdk is 16 so I'm leaning away from using that.
